tests/distributed/test_pynccl.py

```diff
@@ -32,8 +33,7 @@ def update_env(fn):
     # so we need to pass the environment variables as arguments
     # and update the environment variables in the function
     def wrapper(env):
-        import os
-        os.environ.update(env)
```

Comment:
Why don't we just use `os.environ.update`?

Reply:
added warning if some env variables already exist. I think it has better user experience.